### DISCUSSION TITLE
I want install python3 but why still unable? Even i use apt-get update && apt-get upgrade use and still unable. Help me

### DISCUSSION THREAD

**Discussion by sannn1n** [termux/termux-packages#11078] [Q&A] [answered]
![Screenshot_20220625-215256](https://user-images.githubusercontent.com/108186784/175778955-21c45a60-1a13-4f65-945f-139e0ea87dc7.jpg) 
1. apt-get install python

![Screenshot_20220625-214340](https://user-images.githubusercontent.com/108186784/175778788-2d862a3f-4231-4ab2-aaaa-6d4250f45453.jpg) 
2. apt-get upgrade && apt-get upgrade

**Comment by 2096779623** [comment, marked as answer]
https://github.com/termux/termux-packages/issues/6726
